tests/test_time_machine.py

```diff
@@ -656,3 +656,11 @@ def test_fixture_used_twice(time_machine):
 
     time_machine.move_to(EPOCH_PLUS_ONE_YEAR)
     assert time.time() == EPOCH_PLUS_ONE_YEAR
+
+
+def test_monotonic_manual_tick():
+    destination = dt.datetime(2017, 2, 6, 14, 8, 21)
+
+    with time_machine.travel(destination, tick=False):
+        assert time.monotonic() == 0
```

Comment:
Likely there won't be huge consequences here, but this imposes a tighter contract than `time.monotonic` has, and it may in fact be undesirable to always have `time.monotonic()` return `0.0`. It may fit better with some use cases for `time.monotonic()` to just return the same thing as `time.time`, since we're *already* doing away with the "this is a monotonically increasing clock" property _anyway_ if we're letting people set the value of `time.monotonic`.

I think you probably want to test things about the contract instead, like:

```
with time_machine.travel(destination, tick=False):
    t0 = time.monotonic()
    t1 = time.monotonic()
    assert t1 = t0
```

Depending on what you want the interface of `time_machine` to look like, you could also imagine the contract being something like this:

```python
with time_machine.travel(destination):
    t0 = time.monotonic()

with time_machine.travel(destination + dt.timedelta(hours=3)):
    t1 = time.monotonic()

assert t1 - t0 == (dt.timedelta(hours=3) / dt.timedelta(seconds=1))
```

Though I can see why you might not want to have to keep track of that. I could also imagine a different context manager like `time_machine.monotonic_tick` that enforces the relative values of monotonic clocks, where the contract could be like the above _within_ the context manager because of some state within the context, and you raise an exception if the user tries to violate the contract of `time.monotonic`, e.g.:

```python
with time_machine.monotonic_ticks():
    with time_machine.travel(destination):
        time.monotonic()
    
    with pytest.raises(BackwardsMonotonicTickError):
        with time_machine.travel(destination - dt.timedelta(seconds=3)):
            pass
```

(Presumably this last one would be a context decorator, in which case you could easily mark your entire test as "this test must only time travel forwards, never backwards")